bin/cml/runner.js

```diff
@@ -532,11 +533,16 @@ exports.builder = (yargs) =>
         default: '',
         description: 'Specifies the security group in AWS'
       },
-      cloudAwsSubnetId: {
+      cloudAwsSubnet: {
         type: 'string',
         default: '',
         description: 'Specifies the subnet to use within AWS'
       },
+      cloudAwsSubnetId: {
+        type: 'string',
+        default: '',
+        hidden: true
+      },
```

Comment:
should be aliased

Reply:
Yes, but aliases can't be hidden. At least, I haven't found yet how to hide them.

Reply:
I'd rather let the alias show. Simplifies the code a lot. We could always leave out the alias in the online docs.

Reply:
👌🏼